test/test_permissions_api.js

```diff
@@ -0,0 +1,52 @@
+import should from 'should';
+import supertest from 'supertest';
+
+import app from '../src/server';
+import db from '../src/models/db';
+import config from '../src/config';
+
+import { loginAsAdmin } from './common';
+
+const endpointPrefix = '/' + config.get('version');
+const server = supertest.agent(app);
+
+describe('Permissions API', () => {
+  let adminToken;
+  before(function*() {
+    adminToken = yield loginAsAdmin(server);
+  });
+
+  describe('GET ' + endpointPrefix + '/permissions', () => {
+    it('should get list of default permissions', done => {
+      server.get(endpointPrefix + '/permissions')
+            .set('Authorization', 'Bearer ' + adminToken)
+            .expect('Content-Type', /json/)
+            .expect(200)
+            .end((err, res) => {
+              should.not.exist(err);
+              res.body.permissions.should.be.deepEqual(
+                config.get('permissions')
+              );
```

Comment:
nit: I think you can use simply `.expect(config.get('permissions'))` or even `.expect(200, config.get('permissions'))` (see https://github.com/visionmedia/supertest#expectstatus-body-fn) and get rid of the `end` at all with `supertest-as-promised` (an error would give a rejected promise).